bitnami/wavefront-prometheus-storage-adapter/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: wavefront
+  repository: https://charts.bitnami.com/bitnami
+  version: 1.3.1
```

Comment:
I understand this can be a dependency of Wavefront, but not the other way around.
I might want to install Wavefront and use this as sub-chart to transfer the data from Prometheus to Wavefront, I see that use case. But I find weird the opposite thing.

For instance, check ElasticSearch and Kibana: we include Kibana as subchart of Elasticsearch, nor the opposite.

Reply:
I thought the same thing, but it's not the same case. This is actually done in upstream: https://github.com/wavefrontHQ/helm/blob/master/prometheus-storage-adapter/requirements.yaml

And it is only required to deploy Wavefront Proxy, the rest of the components are set to disabled. I added a piece of documentation to clarify that.

Reply:
Fair enough